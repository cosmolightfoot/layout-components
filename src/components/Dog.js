import React from 'react';

const dog = {
  name: 'Spot',
  age: 25,
  weight: '80 lbs'
};

export default function Dog() {
  return (
    <dl>
      <dt>Name</dt>
      <dd>{dog.name}</dd>

      <dt>Age</dt>
      <dd>{dog.age}</dd>

      <dt>Weight</dt>
      <dd>{dog.weight}</dd>
    </dl>
  );
}
