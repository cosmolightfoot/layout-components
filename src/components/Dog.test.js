import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog.js';

describe('Dog test', () => {
  it('renders dog', () => {
    const wrapper = shallow(<Dog />);
    expect(wrapper).toMatchSnapshot();
  });
});
