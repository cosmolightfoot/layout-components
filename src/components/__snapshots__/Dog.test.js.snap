// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog test renders dog 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Spot
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    25
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    80 lbs
  </dd>
</dl>
`;
